helpers/deploymentConfig.ts

```diff
@@ -121,8 +123,8 @@ export const ADDRESSES: PreconfiguredAddresses = {
     frxETH: "0x5e8422345238f34275888049021821e8e08caa1f",
     weETH: "0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee",
     eETH: "0x35fA164735182de50811E8e2E824cFb9B6118ac2",
-    PTweETH: "0xc69Ad9baB1dEE23F4605a82b3354F8E40d1E5966",
-    PTweETHMarket: "0xF32e58F92e60f4b0A37A69b95d642A471365EAe8",
+    PTweETH_26DEC2024: "0xc69Ad9baB1dEE23F4605a82b3354F8E40d1E5966",
+    PTweETH_26DEC2024_Market: "0xF32e58F92e60f4b0A37A69b95d642A471365EAe8",
```

Comment:
We have to use the addresses of the 26DEC2024 token:

```suggestion
    PTweETH_26DEC2024: "0x6ee2b5e19ecba773a352e5b21415dc419a700d1d",
    PTweETH_26DEC2024_Market: "0x7d372819240d14fb477f17b964f95f33beb4c704",
```